test/parallel/test-debug-port-numbers.js

```diff
@@ -44,7 +44,9 @@ function kill(child) {
   try {
     process.kill(-child.pid);  // Kill process group.
   } catch (e) {
-    assert.strictEqual(e.code, 'ESRCH');  // Already gone.
+    // Generally ESRCH is returned when the process group is already gone. On
+    // some platforms such as OS X it may be EPERM though.
+    assert.ok(e.code, 'EPERM' || 'ESRCH');
```

Comment:
I don't think this is right. `'EPERM' || 'ESRCH'` will always evaluate to `EPERM`